docs/src/transforms.md

```diff
@@ -353,13 +362,13 @@ This transform will take a query, serialise it into a RESP2 compatible format an
 
     # When this field is provided TLS is used when connecting to the remote address.
     # Removing this field will disable TLS.
-    tls:
-      # Path to the certificate file, typically named with a .crt extension.
-      certificate_path: "tls/redis.crt"
-      # Path to the private key file, typically named with a .key extension.
-      private_key_path: "tls/redis.key"
-      # Path to the certificate authority file typically named ca.crt.
-      certificate_authority_path: "tls/ca.crt"
+    #tls:
+    #  # Path to the certificate authority file, typically named ca.crt.
+    #  certificate_authority_path: "tls/ca.crt"
```

Comment:
Should the paths be different between examples? i.e. do all the paths make sense when uncommented?

Reply:
oh yeah those path prefixes are a bit of mess, I'll tidy that up a bit.

Reply:
@XA21X @conorbros Please check the latest commit and let me know what you think of that change.
The file names are left as is because they reflect the naming used by the example key generator for redis and cassandra respectively but the prefix is normalized because there is no need to care about the shotovers internal example/test key paths.

Reply:
LGTM, thanks for the cleanup.

Reply:
LGTM